src/translib/transformer/xfmr_intf.go

```diff
@@ -1816,8 +1860,8 @@ var populateMGMTPortCounters PopulateIntfCounters = func (inParams XfmrParams, c
     stats := strings.Fields(entry)
     log.Info(" Interface filds: ", stats)
 
-    ret := getMgmtSpecificCounterAttr(targetUriPath, stats, counter)
-    log.Info(" getMgmtCounters : ", *counter)
+    ret := getMgmtSpecificCounterAttr(targetUriPath, stats, counter_val)
+    log.Info(" getMgmtCounters : ", *counter_val)
```

Comment:
counter_val can be nil right, if we try to print, it can cause panic.

Reply:
Counter_val is built by the caller, we are just populating attributes and dumping as part of this log.